File: app/src/main/java/com/internaltest/sarahchatbotmvp/models/Message.java
package com.internaltest.sarahchatbotmvp.models;

public class Message {

  private final String message;
  private final boolean isReceived;

  public Message(String message, boolean isReceived) {
    this.message = message;
    this.isReceived = isReceived;
  }

  public String getMessage() {
    return message;
  }

  public boolean getIsReceived() {
    return isReceived;
  }

}

File: app/src/main/java/com/internaltest/sarahchatbotmvp/models/Message.kt
package com.internaltest.sarahchatbotmvp.models

import com.google.firebase.firestore.PropertyName

data class Message(
    val message: String = "",
    @get:PropertyName("received")
    @set:PropertyName("received")
    var isReceived: Boolean = false,
    var data : String = ""
) {
    //não deletar
    constructor() : this("", false)
}